app/helpers/wizard/event.py

```diff
@@ -402,3 +405,12 @@ def save_resized_background(background_image_file, event_id, size, image_sizes):
     )
 
     return save_resized_image(background_image_file, width_, height_, basewidth, aspect, height_size, upload_path)
+
+
+def save_social_links(social_links, event):
+
+    for social_link in social_links:
+        social_exists = db.session.query(SocialLink).filter_by(name=social_link['name'], link=social_link['link'], event_id=event.id).scalar()
```

Comment:
Use the SQLAlachemy ORM syntax that we are using throughout the application like `SocialLink.query.filter_by` .. What you have done is also correct but let's follow the same syntax throughout ...